sql/check_sequence_overflow.sql

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2019-2024. Ivan Vakhrushev and others.
+ * https://github.com/mfvanek/pg-index-health-sql
+ *
+ * Licensed under the Apache License 2.0
+ */
+
+select
+    schemaname, -- schema name
```

Comment:
You need to rewrite this query according to general rules https://github.com/mfvanek/pg-index-health-sql/blob/master/CONTRIBUTING.md